improving.html

```diff
@@ -154,14 +154,14 @@ <h2 id="scope">What is your Scope?</h2>
 
 	<h2 id="target">What is your Target Level of Accessibility?</h2>
 	<p>The generally accepted target for accessibility is <a href="https://www.w3.org/WAI/intro/wcag">Web Content Accessibility Guidelines (WCAG) 2.0</a> Level AA. This may already be the standard specified in your organizational policy or it may be the legal requirement for your website.</p>
-	<p>You may need to define a phased approach with different dates for different levels. For example, meet particular WCAG 2.0 success criteria in the next release, and meet all Level A and Level AA success criteria in the following release.</p>
-	<p>Note that in some cases, some Level AAA success criteria may be fairly easy to meet. For example, refining appropriate link text (2.4.9, Level AA) and heading structure (2.4.10, Level AAA) may be easy to address together when revising content.</p>
+	<p>You may need to define a phased approach with different dates for different levels. For example, meet particular WCAG 2.0 success criteria in the next release, and meet all <a href="https://www.w3.org/WAI/WCAG20/quickref/?currentsidebar=%23col_overview&levels=aaa">Level A and Level AA</a> success criteria in the following release.</p>
```

Comment:
* [Editor’s Discretion] Add link to Quickref , only selecting A+AA (yeah, talk to the developer about the bad notation for the levels parameter showing what is NOT selected… 😔)